src/specs/models/api-body-schemas/PlotTableConfig.v1.yaml

```diff
@@ -12,13 +12,13 @@ properties:
     type: string
     description: The type of the plot to be saved. Can be one of the values specified herein.
     enum:
-      - umisInCells
-      - umisCellRank
-      - mitochondrialContent
-      - mitochondrialReads
-      - classifierContour
-      - genesVsUmisHistogram
-      - genesVsUmisScatterplot
+      - cellSizeDistributionHistogram
+      - cellSizeDistributionKneePlot
+      - mitochondrialFractionHistogram
+      - mitochondrialFractionLogHistogram
+      - classifierEmptyDropsPlot
+      - featuresVsUMIsHistogram
+      - featuresVsUMIsScatterplot
```

Comment:
Does this plot type have to have a unique value? If not, I was thinking that it might be better to have the plot type can be named without the filter name in it, for example "histogram", "logHistogram". In that way, in the future it will be easier to integrate more plots of types that we already support, instead of having to update this schema every time when adding a new plot. For example, it will be very useful for the "custom data processing pipelines" work that we might end up doing (see this slack thread: https://this-is-biomage.slack.com/archives/C01CP46TJ0Z/p1615913630008800). What do you think? Is it too much of refactoring to be worrying about this now?

Reply:
Yea, this was one the issues that I presented to you with the slide. Right now there is no way to point which plot we mean if we use a common component. For example, we can create a common plot type, e.g. `histogram`, but there is no way to identify which histogram yet, is it at the cell size distribution, mitochondrial fraction or genes vs UMIs? So we resort to giving each plot a unique name.

In the long run, we have to look at how we deliver plots.

Reply:
yes, sounds like we have to go with that one for the MVP, but after that we will need to revisit this